docs/admin.rst

```diff
@@ -52,15 +52,25 @@ Upgrade Docs`_ for further details on how to update the suggested firewall.
 .. _`Netgate RSS Feed`: https://www.netgate.com/feed.xml
 .. _`pfSense Upgrade Docs`: https://doc.pfsense.org/index.php/Upgrade_Guide
 
-Keep your Tails Drives Updated
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Keep your SecureDrop Workstations Updated
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You should keep your SecureDrop workstations updated with:
+
+* **Tails updates**: for each *Admin Workstation*, *Journalist Workstation*, and
+  *Secure Viewing Station*
+* **SecureDrop workstation updates**: for each *Admin Workstation* and
+  *Journalist Workstation*
 
-You should apply updates to your Tails drives as they are released, as they
-can contain critical security fixes. Subscribe to the `Tails RSS Feed`_ to be
+You should apply Tails updates to your Tails drives as they are released, as they
+often contain critical security fixes. Subscribe to the `Tails RSS Feed`_ to be
```

Comment:
Out of scope of this PR, but what are you thoughts about this:
 `check_for_update` could run `/usr/local/bin/tails-version` and parse the running version, and we could store the latest version in a constant somewhere.

Reply:
Clever! Running `/usr/local/bin/tails-version` might come in handy, e.g. if we know that a certain Tails release series no longer works well with our current workstation logic, we could display a message gently encouraging the user to upgrade Tails